api/overview/azure/latest/event-hubs.md

```diff
@@ -6,43 +6,43 @@ ms.author: pallavit
+
+### Version 5.x
+
+The version 5.x client libraries are part of the Azure SDK for .NET. The source code for the Azure Event Hubs client libraries is available on [GitHub](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub).
+
+Use the following packages to publish and consume events from Event Hubs:
+
+| NuGet Package | Reference | Samples |
+|--------------------------------------|---------------------------------------------------------------|-------------------------------------------------------------------------------|
+| [Azure.Messaging.EventHubs](https://www.nuget.org/packages/Azure.Messaging.EventHubs)  |  [API Reference for Azure.Messaging.EventHubs](/dotnet/api/azure.messaging.eventhubs)  |  [Samples for Azure.Messaging.EventHubs](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs/samples)  |
+| [Azure.Messaging.EventHubs.Processor](https://www.nuget.org/packages/Azure.Messaging.EventHubs.Processor)  |  [API Reference for Azure.Messaging.EventHubs.Processor](/dotnet/api/azure.messaging.eventhubs)  |  [Samples for Azure.Messaging.EventHubs.Processor](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples)  |
+
+### Version 4.x
+
+The source code for version 4.x of the Azure Event Hubs client libraries is available on [GitHub](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub).
+
+Use the following legacy packages to publish and consume events from Event Hubs:
+
+| NuGet Package | Reference | Samples |
+|--------------------------------------|---------------------------------------------------------------|-------------------------------------------------------------------------------|
+| [Microsoft.Azure.EventHubs](https://www.nuget.org/packages/Microsoft.Azure.EventHubs)  |  [API Reference for Microsoft.Azure.EventHubs](/dotnet/api/overview/azure/eventhubs/v4)  |  [Samples for Azure.Messaging.EventHubs](https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/)  |
+| [Microsoft.Azure.EventHubs.Processor](https://www.nuget.org/packages/Microsoft.Azure.EventHubs.Processor)  |  [API Reference for Microsoft.Azure.EventHubs.Processor](/dotnet/api/microsoft.azure.eventhubs.processor)  |  [Samples for Azure.Messaging.EventHubs.Processor](https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/)  |
+
+## Libraries for resource management
+
+Use the following library to work with the Azure Event Hubs resource provider:
+
+| NuGet Package | Reference |
+|--------------------------------------|---------------------------------------------------------------|
+| [Azure.ResourceManager.EventHubs](https://www.nuget.org/packages/Azure.ResourceManager.EventHubs)  | [API Reference for Azure.ResourceManager.EventHubs](/dotnet/api/overview/azure/resourcemanager.eventhubs-readme)  |

```

Comment:
This link does not exist in the [source](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/README.md).   How is this getting translated?

//cc: @JimSuplizio, @danieljurek

Reply:
These tables are hand-produced and need to be updated manually by the team. We do produce and update a list of packages associated with the service here: https://github.com/Azure/azure-docs-sdk-dotnet/blob/main/api/overview/azure/latest/event-hubs-index.md 


That can be included in this overview using an `[!INCLUDE...` directive like the one here: https://github.com/Azure/azure-docs-sdk-dotnet/blob/main/api/overview/azure/latest/active-directory.md?plain=1